databases.qmd

```diff
@@ -607,7 +586,7 @@ flights |>
   )
 ```
 
-The translation of summary functions becomes more complicated when you use them inside a `mutate()` because they have to turn into a window function.
+The translation of summary functions becomes more complicated when you use them inside a `mutate()` because they have to turn into a so-called **window** function.
```

Comment:
```suggestion
The translation of summary functions becomes more complicated when you use them inside a `mutate()` because they have to turn into so-called **window** functions.
```